hardware/asic/skywater/Makefile

```diff
@@ -0,0 +1,7 @@
+#Makefile for OpenLane and OpenRAM started on 2/11/21
+SRAM:
+	python3 $(OPENRAM_HOME)/openram.py $(OPENRAM_CONFIG)/myconfig.py
+	
+.PHONY clean:
```

Comment:
I am not aware of this syntax. Are you sure?

I use 
.PHONY: clean 

let's start putting the phony list before the targets 

there is no target clean, so  pls create it after you declare it as phony

Reply:
Sir I am making all these changes now

Reply:
Kindly check now